discoverx/msql.py

```diff
@@ -67,7 +67,10 @@ def compile_msql(self, table_info: TableInfo) -> list[SQLRow]:
         for classified_cols in col_class_combinations:
             temp_sql = msql
             for classified_col in classified_cols:
-                temp_sql = temp_sql.replace(f"[{classified_col.class_name}]", classified_col.name)
+                # Unescaped column name
```

Comment:
I am probably missing something here, but why do we need to use square brackets at all here? For the temp_sql we are simply replacing all of them. Why can't we use the correct sql syntax directly and use backticks where needed?

Reply:
Yes, we can also do that.
It's just that it's easy to forget to add them every time we create a new query (they are needed 90% of the times).